<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PRAChecker</class>
 <widget class="QMainWindow" name="PRAChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PRAChecker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="pb_status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>411</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_check_sample">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Check New Samples</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>417</width>
      <height>112</height>
     </rect>
    </property>
    <property name="title">
     <string>Delta Check Setting</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="5" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Fold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Fold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Fold</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="7">
      <widget class="QSpinBox" name="spin_box_min_ad_class1"/>
     </item>
     <item row="0" column="4" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QSpinBox" name="spin_box_ad_class1"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_box_ad_control"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_box_ad_class2"/>
       </item>
      </layout>
     </item>
     <item row="0" column="6" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Minimum AD:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Minimum AD:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Minimum AD:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="7">
      <widget class="QSpinBox" name="spin_box_min_ad_control"/>
     </item>
     <item row="0" column="0" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Mean Class I MFI:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Mean  Control MFI:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Mean  Class II MFI:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="7">
      <widget class="QSpinBox" name="spin_box_min_ad_class2"/>
     </item>
     <item row="0" column="1" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSpinBox" name="spin_box_mean_class1"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_box_mean_control"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_box_mean_class2"/>
       </item>
      </layout>
     </item>
     <item row="0" column="3" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Class I AD:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Control AD:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Class II AD:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>advanced settings</string>
    </property>
    <widget class="QCheckBox" name="checkbox_ks_testing">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kolmogorovâ€“Smirnov test  if AD exceed</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>42</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Class I:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Control:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Class II:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>41</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QSpinBox" name="spin_box_min_ad_ks_class1"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spin_box_min_ad_ks_control"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spin_box_min_ad_ks_class2"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>k-S test</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>21</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>41</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QSpinBox" name="spin_box_ks_class1"/>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="spin_box_ks_control"/>
      </item>
      <item row="2" column="0">
       <widget class="QSpinBox" name="spin_box_ks_class2"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="menu_folder_monitor"/>
    <addaction name="separator"/>
    <addaction name="menu_database_rebuilder"/>
    <addaction name="separator"/>
    <addaction name="menu_ouput_folder"/>
    <addaction name="separator"/>
    <addaction name="menu_acquire_FCS_Files"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_folder_monitor">
   <property name="text">
    <string>Folder To Monitor</string>
   </property>
  </action>
  <action name="menu_database_rebuilder">
   <property name="text">
    <string>Database Rebuild</string>
   </property>
  </action>
  <action name="menu_ouput_folder">
   <property name="text">
    <string>Output Folder</string>
   </property>
  </action>
  <action name="menu_acquire_FCS_Files">
   <property name="text">
    <string>Acquire FCS Files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
